Fredmanzi/EPIC-MAIL
# #164138561 built signup page

#### What does this PR do?
creates a signup page 
#### Description of Task to be completed?
1.create the fields for the new  user to create an account eg name, email, password fields
2.create button to submit the user credentials 
#### How should this be manually tested?
when the user navigates to signup page he should see a responsive form to provide his/her credentials and submit button to submit.
